training/src/main/java/wolox/training/models/User.java

```diff
@@ -34,6 +33,9 @@
     @NotNull
     private String username;
 
+    @NotNull @Column(length = 60)
+    private String password;
```

Comment:
Password will get serialized like this whenever someone "gets" a user.
Find out how to specify that this property should be writtable only ;)